### PR TITLE
Websockets, SSL, travis testing

### PR DESCRIPTION
**Pull request by zbyte64**
Added features:

- Websocket to container
- Support SSL
- Travis CI
- Read docker connection info from environment
- Fix log streams and parsing of log lines in newer versions of docker
- Add `timeout` keyword arg for container `wait`
- Added example of send & receiving stdio over a websocket

### CODE REVIEW

**Comment by zbyte64** [comment]
Just added support for streaming response from image pull requests, that addresses #3
...And support for `container.put_archive`

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/zbyte64/aiodocker.svg?branch=master)](https://travis-ci.org/zbyte64/aiodocker)
```
Comment (paultag) [inline README.md]: (I'll move this to mine when I merge)

**Comment by paultag** [comment]
Looks great so far, thanks @zbyte64 -- after I land this, if you're interested in co-maintaining in, id love the help! :D

**Comment by zbyte64** [comment]
Cool, I can definitely help maintain this project.

**Comment by paultag** [comment]
Thanks, @zbyte64! I'll add you as a collab, thanks for all this, this is awesome. :heart:

**Pull request merged**